### PR TITLE
made some additional changes

### PR DESCRIPTION
**Pull request by lilvina**

### CODE REVIEW

File: src/scrab-bag.js [hunk 1]
```diff
@@ -1,41 +1,111 @@
...
+  9: ['I'],
+  8: ['A'],
+  7: ['O'],
+  5: ['R', 'N', 'T'],
+  4: ['U', 'L', 'D'],
+  3: ['G', 'S'],
+  2: ['P', 'H', 'F', 'V', 'W'],
+  1: ['B', 'Y', 'J', 'K', 'M', 'Q', 'C', 'Z', '_'],
+  0: ['X']
+}
```
Comment (rae-ralston) [inline src/scrab-bag.js:42]: could probably delete example output? this isn't actually used, it was just for reference?

File: src/scrab-bag.js [hunk 2]
```diff
@@ -1,41 +1,111 @@
...
-  }
+  let usedTilesArray = input.toUpperCase().split(' ')
 
-  let array = input.toUpperCase().split(' ')
-  for(let i = 0; i < array.length; i++) {
-    let currentLetters = array[i]
-    let remainderPieces = currentLetters[array]
-    remainderPieces = remainderPieces - 1
+  for(let i = 0; i < usedTilesArray.length; i++) {
+    if(usedTilesArray[0] in TILES_TOTALS && TILES_TOTALS.usedTilesArray[i] < 0) {
```
Comment (rae-ralston) [inline src/scrab-bag.js:48]: if you have time later check if both of these are needed.

File: src/scrab-bag.js [hunk 3]
```diff
@@ -1,41 +1,111 @@
...
+      TILES_TOTALS.usedTilesArray[i] = TILES_TOTALS.usedTilesArray[i] - 1
+    }
+    return exampleOutput
```
Comment (rae-ralston) [inline src/scrab-bag.js]: Is this what you came and told me about? We shouldn't be returning this yah?

File: src/scrab-bag.js [hunk 4]
```diff
@@ -1,41 +1,111 @@
...
+//     D: 4,
+//     E: 12,
+//     F: 2,
+//     G: 3, 
+//     H: 2,
+//     I: 9,
+//     J: 1,
+//     K: 1,
+//     L: 4,
+//     M: 2,
```
Comment (rae-ralston) [inline src/scrab-bag.js]: take out comments for PR's

**Review by rae-ralston — COMMENTED** [review]
Clap

File: src/scrab-bag.js (hunk 3 above)
Comment (lilvina) [reply src/scrab-bag.js]: Yes. It kept returning the input but not the output.

**Pull request merged**